public_html/launch.php

```diff
@@ -324,8 +324,8 @@ function build_form_param($param_id, $param, $is_required){
     // Boolean input
     if($param['type'] == 'boolean'){
         $input_el = '
-        <div class="form-control pl-4">
-            <div class="form-check form-check-inline mr-4">
+        <div class="form-control pl-md-4">
+            <div class="form-check form-check-inline  mr-md-4">
```

Comment:
Does this work with the version of bootstrap that we’re using?

Reply:
oops, no, it doesn't. But I then actually don't think the padding/margin is necessarily needed. It mostly takes space away for narrow screens, while not really adding much visual balance in wider screens, imo.

On a related note: the radio buttons look in general a bit off to me in the current layout. What do you think about using a switch element instead?

Reply:
I wondered about a switch when I was writing it, but I thought that it was slightly less explicit. I quite liked having the values that will be set written out.

When I had seen this overflow before I had assumed that I would move things onto two lines for mobile. Would that be better than trying to squish everything into a tiny space? Might be more future-proofed too, as we don’t know how long all parameters will be.

I also have a vague memory of giving the parameter names scrolling x-overflow so that they don’t overlap the form controls. Is this not the case?

Reply:
For the parameters I saw, "On/Off" would make equal sense instead of "True/False", that's why I think a switch would translate quite well and even be a bit more intuitive/easier to parse.

Reply:
Yes maybe, can we test it and see how it looks?